Governance/publish/publish-procedure.md

```diff
@@ -0,0 +1,97 @@
+# Publishing procedure
+
+While the exact procedure to publish a dataset will depend on where the data is published, there are common steps to prepare.
+
+````{grid} 1 1 1 3
+:class-container: text-center
+:gutter: 3
+
+```{grid-item-card}
+:class-header: bg-light
+
+Step 1: Choices
+^^^
+The first step is to make some important choices, which will depend on the characteristic of the data, the project that produced it and its most likely use.
+```
+
+```{grid-item-card}
+:class-header: bg-light
+
+Step 2: Metadata
+^^^
+Metadata includes any information on the data itself. In a published record the metadata is usually available as a record abstract and inside the files (if using a self-describing format as netCDF) or by auxiliary metadata files.
+Ideally all this combined information should allow someone to reproduce the data from scratch (see [Provenance](../concepts/provenance)).
+```
+
+```{grid-item-card}
+:class-header: bg-light
+
+Step 3: Formatting files
+^^^
+Files should be well formatted to enhance their accessibility and usability.
+```
+````
+
+`````{tab-set}
+````{tab-item} Choices    
+   :class-container: px-100 
+**What** 
+
+It's not usual or a good thing to publish all the data produced in a research project. It's important to identify what data is useful or required, considered other limitations like storage availability. Here are some considerations that can help with the choice.
+1. Provide the information needed to interpret, reuse and reproduce the results. This is what journal publishers usually require, most of them provides [guidelines and examples](journals) of which data should be shared.
+ 
+2. If the output is big publish only a subset. If methods are well described, the software used is easily available, then publishing only the subset of data that underlines a publication is sufficient. For example, the post-processed output is sufficient for a model simulation. However, the model version and configuration used, the input data and model source code should be documented.
+ 
+3. It's essential to consider an end user point of view. What would a user look for when considering using a dataset? What kind of information is essential for the data to be usable? Which additional information would make its use easier?
```

Comment:
Good point, I was going to also add something like that in the abstract part which I haven't tackle yet, as giving context to the data helps highlighting that too, but it's worth to make it explicit

Reply:
Re-reading this I might move the current point 3 to the "abstract" I haven't wrote yet because it's something to consider in the data description and make your suggestion point 3. And then I would also mention that in the abstract

Reply:
3. Consider adding an example of what's essential to consider from an end-user point of view. For example, ACS aims to publish bias-corrected data on a 5km grid; this decision is mainly based on user needs who want to have higher and higher resolved data. But I am not sure if that's an example for data creation rather than publish procedure.

Reply:
I've made this changes pushing now, I also change the abstract part to a more comprehensive  "dataset web page" to include keywords, related works , tile etc.

Reply:
sorry Alice I missed your comment before, yes probably that's nearly going into data product while here is more about what to choose among what's available, rather than making a decision at the start of the project on what the output should look like.